hooks/usePortfolioOverview/index.tsx

```diff
@@ -58,8 +58,8 @@ export const usePortfolioOverview = (): PortfolioOverview => {
                     .plus(calcNotionalValue(longTokenPrice, userBalances.aggregateBalances.longTokens))
                     // TODO handle non stable coin settlementTokens
                     .plus(userBalances.aggregateBalances.settlementTokens)
-                    .plus(pendingAmounts.longMint.div(nextLongTokenPrice))
-                    .plus(pendingAmounts.shortMint.div(nextShortTokenPrice))
+                    .plus(pendingAmounts.longMint)
+                    .plus(pendingAmounts.shortMint)
```

Comment:
I believe we still want the div here. Since this is total portfolio value we want to add the expected value of the pool tokens being minted instead of just the amount of settlement tokens being spent